### PR TITLE
fix: Bump up appshell & Simplify usage with PluginComponents

### PR DESCRIPTION
**Pull request by fkolar**
Removed this concept and got back to angular services.

### CODE REVIEW

File: packages/content-item-app/src/app/your-favorites/your-favorites.component.ts [hunk 1]
```diff
@@ -1,47 +1,34 @@
...
   ngOnInit(): void {
-  }
 
-  getConfiguration(): Partial<PluginConfiguration> {
-    const event = this.appEvent;
-    return {
-      addListeners: function (): Array<Listener> {
-        return [event]
+    this.subscriber = this._msg.subscribe(TOPIC_APP_EVENT, (m => {
+      if (m instanceof TextMessage) {
```
Comment (valorkin) [inline packages/content-item-app/src/app/your-favorites/your-favorites.component.ts:22]: filter before subscribe will look better, but it's optional

File: packages/content-req-app/src/app/pr-list/pr-list.component.ts [hunk 2]
```diff
@@ -1,35 +1,18 @@
...
   templateUrl: './pr-list.component.html',
 })
-export class PrListComponent implements OnInit, PluginComponent {
+export class PrListComponent implements OnInit, OnDestroy {
   tableRows: any[];
-  private messaging: MessagingService;
+  subscriber: TopicSubscriber<Message>;
+
+  constructor(private messaging: MessagingService) {
+    this.messaging = messaging;
```
Comment (valorkin) [inline packages/content-req-app/src/app/pr-list/pr-list.component.ts]: access modifier will do it for you

**Review by valorkin — DISMISSED** [review]
couple of minor comments

File: packages/content-item-app/src/app/your-favorites/your-favorites.component.ts (hunk 1 above)
Comment (fkolar) [reply packages/content-item-app/src/app/your-favorites/your-favorites.component.ts:22]: right like to make it more functional. well next version..

File: packages/content-req-app/src/app/pr-list/pr-list.component.ts (hunk 2 above)
Comment (fkolar) [reply packages/content-req-app/src/app/pr-list/pr-list.component.ts]: Yeah, this is left over,  as I had exising member that I was not settings from contructor. we dont need this anymore

**Review by valorkin — APPROVED** [review]

**Pull request merged**